infrastructure/modules/serverless/cloudtrail.tf

```diff
@@ -15,16 +15,18 @@
 # Create an S3 bucket for storing CloudTrail logs
 resource "aws_s3_bucket" "this" {
   #ts:skip=AWS.S3Bucket.LM.MEDIUM.0078 We are aware of the risk and choose to skip this rule
+  count = var.create_kms ? 0 : 1
+
   bucket = "${var.namespace}-cloudtrail-logs-storage"
   tags   = merge(local.tags, tomap({ "Name" = "${var.namespace}-bucket" }))
 
 }
 
-resource "aws_s3_bucket_acl" "this" {
-  bucket = aws_s3_bucket.this.id
-  acl    = "private"
-  #   depends_on = [aws_s3_bucket_ownership_controls.s3_bucket_acl_ownership]
-}
+# resource "aws_s3_bucket_acl" "this" {
```

Comment:
Why are we commenting this code, do we need it or just commenting it out. If we don't need this you can remove it

Reply:
Removed